.gitignore/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Kindly ensure the file structure of your linter files is correct, it's supposed to be .github/workflows/linters.yml. Kindly also unmerge your commits so that the reviewer can check if the linters are passing.